client/src/template/SideBar/SideBar.tsx

```diff
@@ -26,26 +49,14 @@ const SideBar: FC<SodeBarProps> = ({ className }) => {
 			</Link>
 
 			<nav className={classes.nav}>
-				<NavLink to={_URL.MEETINGS} className={handleActiveClass}>
-					зустрічі
-				</NavLink>
-				<NavLink to={_URL.FORM_FEEDBACK} className={handleActiveClass}>
-					відгук по зустрічі
-				</NavLink>
-				<NavLink to={_URL.FORM_SENKAN} className={handleActiveClass}>
-					сєнкан
-				</NavLink>
-				<NavLink to={_URL.HOMEWORKS} className={handleActiveClass}>
-					домашки
-				</NavLink>
-				<NavLink to={_URL.PAYMENTS} className={handleActiveClass}>
-					оплати
-				</NavLink>
+				{mappedNavLinks}
 
 				<div className={classes.usefullLinks}>
-					<NavLink to={_URL.MONO} target="_blank" className="grow text-center border-rose-400">
-						💸 скинуть баблішко 💸
-					</NavLink>
+					{user.data?.data?.role === 'student' && (
```

Comment:
you have `ROLES` enum in `@/types`
consider using it for each and every case related to roles